hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -548,6 +557,10 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
                       <Button>Display Limit: {displayLimit}</Button>
                     </Dropdown>
                   </div>
+                  &nbsp;&nbsp;&nbsp;&nbsp;
+                  <Tooltip placement="rightTop" title="Additional block size added for viewing small entity sizes. Please refer to pie-chart tooltip for exact size information.">
```

Comment:
```suggestion
                  <Tooltip placement="rightTop" title="Additional block size is added to each entity, so even the small entities are visible. Please refer to pie-chart tooltip for exact size information.">
```

Reply:
@dombizita 
Done changes in latest commit and added new screenshot with tooltip.